### PR TITLE
No blood, feasts or DNA from NPC Humans

### PR DESCRIPTION
**Pull request by flappybatpal**
[WIKI][BALANCE]

## About the PR 
Prevents targeting NPC humans for Changeling absorb, Werewolf feast or Vampire blood abilities.

## Why's this needed? 
Robust players will start the round by killing 4+ NPCs.

## Changelog

```changelog
(u)Flappybat
(*)Human NPCs are no longer valid targets for Changeling's DNA absorb, Werewolf feast and Vampire blood abilities.
```

### CODE REVIEW

File: code/mob/living/carbon/human/gimmick.dm [hunk 1]
```diff
@@ -347,6 +349,7 @@ proc/empty_mouse_params()//TODO MOVE THIS!!!
 /mob/living/carbon/human/biker
 	real_name = "Shitty Bill"
 	gender = MALE
+	is_npc = 1
```
Comment (DimWhat) [inline code/mob/living/carbon/human/gimmick.dm:352]: `is_npc = TRUE`

**Comment by wolfolotl** [comment]
against this. as an RP server regular, it's a huge relief to have a way to get abilities for a potential antag gimmick without having to potentially cut someone else's round short. plus this would remove some pretty cool emergent gameplay on the off chance that someone catches you in the act

**Comment by wolfolotl** [comment]
> Honestly that's a really good reason to merge this, because avoiding conflict on RP as an antag with an easy meta way to gain powers is the exact opposite of good RP and what an antag should be doing.
I mean I guess but also like, I don't wanna differentiate between people who I have to kill fast just to get some dna and people whose deaths I can draw out because I'm scaring them with my freaky noodle arm. feels bad to do that feels metafriendy

**Comment by flappybatpal** [comment]
Well I'm also not too fond of "Oh all the diner NPCs are dead/missing so there is a vamp/changeling/werewolf around" plus just the fact of the diner NPCs missing within ten minutes in half the game modes.

**Comment by MetricDuck** [comment]
Could just bump up the points people start off with, given the reason stated for in "Why's this needed?" is that robust players are starting off with 4 NPCs worth anyway.

**Comment by nefarious6th** [comment]
dunno how admins feel about potential for increased ahelps and such to waffle's point about metafriend-y stuff that might result.

i wouldnt be against this. i think the level of effort one should put in for early player kills should be just as high as what they're doing for later-game kills. i see no reason why there needs to be a sort of shame or divergence away from potentially just spending all round interacting with 2 or 3 players and doing serious and good rp with them versus using them as little chips to speed along to a higher-DNA goal. make the low-DNA stages of changeling fun, get good banter going in the hivemind, save DNA for limb friends versus aboming. and if the DNA objectives of 6 or 8 or 10 people seem to push people too much in the direction of speeding through the first 4 people, we can always lower the DNA total objectives on RP again.

**Comment by Momoberry** [comment]
I'm for it, personally. the diner rush strat is done to death and it's honestly just kind of annoying having to deal with people rushing it ASAP, cause it also makes pointing them out uncomfortably easy, but I just have to pretend like I don't know.

**Comment by DimWhat** [comment]
I like this having played classic a fair amount. If people feel like there is an issue with how these antagonists are on RP without these extra points, there might be a bigger issue.

**Comment by Azrun** [comment]
Yes! More intrigue, you have control of how long you spend planting seeds of trust, confusion, and lies to reap the benefits.  Tension!  

Narratively, running off station to get a snack from NPCs that are easily overpowered doesn't really make a whole lot of sense...

Also there are options for players to come back on RP, I understand that might not be as satisfying having to shift gears but it helps move the round forward mechanically to a round ending scenario or the elimination of the crews threat(s).

**Review by Tarmunora — APPROVED** [review]

**Comment by Chatauscours** [comment]
I am somewhat against it, at least on RP. 
Some people just want to do some funny gimmick with their changeling abilites without killing any crew, but with this PR such gimmicks can and will be shut down by security as they find the murderer of that one staffie.

**Comment by Bartimeus973** [comment]
In my opinion, i think this is overall a good change.
Pushing the antagonists to earn their points through inter-player conflict will lead to more interesting action especially on the RP servers. It was nice to have the option to get some points very early on without killing someone, but losing this option in favor of pushing antags to escalate more and cause more interesting conflict is worth it. Most non-lethal abilities used for RP are unlocked by default on vampire and changeling anyhow so this shouldnt prove to be much of an issue. Additionally, vampires and changelings had the option of being able to use most of their powers very early in the shift with no real counterplay aside from rushing the space diner to check for an antag, which isnt fun for anyone and extremely metagamey. Now if a changeling becomes a shambler, or a vampire pulls out frost bats, we'll know it was earned and didnt come from eating monkeys/npcs in a corner alone for awhile.

**Comment by geneva-breaker** [comment]
absolutely bewildering, wow

**Pull request merged**